src/test/java/config_loaders/EnvironmentConfigLoader.java

```diff
@@ -0,0 +1,43 @@
+package config_loaders;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class EnvironmentConfigLoader {
+    private Properties properties;
+
+    public EnvironmentConfigLoader() {
+        properties = new Properties();
+
+        try {
+            String projectPath = System.getProperty("user.dir");
+
+            InputStream inputStream = new FileInputStream(projectPath + "/src/main/resources/environment.properties");
+            properties.load(inputStream);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public WebDriver getDriver() {
+        if (properties.getProperty("browser").equals("chrome")){
+            return new ChromeDriver();
+        }
+        if  (properties.getProperty("browser").equals("firefox")){
+            return new FirefoxDriver();
+        }
+        else {
+            //what if I got no browser in properties? how should I deal with it?
```

Comment:
It's up to you. You can consider properties file as invalid and throw RunTimeError or use some default value, e.g. 'firefox'.